src/app/authentication/sign-up/sign-up.component.ts

```diff
@@ -62,7 +63,7 @@ export class SignUpComponent implements OnInit {
                     }, hideSpinnerAndDisplayError);
             }, err => {
                 SpinnerUtil.hideSpinner();
-                swal('', err.error.message, 'warning');
+                this.popupService.warning(err.errror.message);
```

Comment:
```suggestion
                this.popupService.warning(err.error.message);
```

This happens, but try to review it by yourself.